### PR TITLE
ad4IoT auth parser

### PR DESCRIPTION
**Pull request by oshezaf**

### CODE REVIEW

**Review by YaronFruchtmann — COMMENTED** [review]
see comments inline

File: Parsers/ASimProcessEvent/ProductParsers/ProcessEventMicrosoftAD4IoT.yaml [hunk 1]
```diff
@@ -35,24 +35,23 @@ ParserQuery: |
...
+                  DvcHostname = DeviceId // OK
+                 | extend
                   DvcOs = iif (EventDetails.MessageSource == "Linux", "Linux", "Windows") // Intermediate fix
                 | extend
                   TargetUsernameType = iif (DvcOs == "Windows", "Windows", "Simple"), 
                   TargetUsername = iff (DvcOs == "Windows", tostring(EventDetails.UserName), ""), 
                   ActingProcessId = iff (DvcOs == "Windows", tostring(EventDetails.ParentProcessId), "") 
                 | project-rename
-                  // EventProductVersion = AgentVersion, // Not available in Windows
+                  EventProductVersion = AgentVersion, // Not available in Windows
```
[the remark this answers is not in the document]
Comment (oshezaf) [reply Parsers/ASimProcessEvent/ProductParsers/ProcessEventMicrosoftAD4IoT.yaml:54]: Modofying

File: Parsers/ASimAuthentication/ProductParsers/AuthenticationMicrosoftAD4IoT.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+Parser:
```
[the remark this answers is not in the document]
Comment (oshezaf) [reply Parsers/ASimAuthentication/ProductParsers/AuthenticationMicrosoftAD4IoT.yaml:1]: Those that are not needed. I did leave a few.

File: Parsers/ASimAuthentication/ARM/AuthenticationMicrosoftAD4IoT/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Azure Defender 4 IoT ASIM Authentication Normalization Parser
```
[the remark this answers is not in the document]
Comment (oshezaf) [reply Parsers/ASimAuthentication/ARM/AuthenticationMicrosoftAD4IoT/README.md:1]: Yes. I am anyway waiting for official naming from Idan, though if I don't get them, I will publish anyways.

File: Parsers/ASimAuthentication/ProductParsers/AuthenticationMicrosoftAD4IoT.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  Title: ASIM Azure Defender for IoT Authentication Parser
+  Version: '0.1.0'
+  LastUpdated: Aug 16, 2021
+Product:
+  Name: Azure Defender for IoT
+Normalization:
+  Schema: ProcessEvent
```
[the remark this answers is not in the document]
Comment (oshezaf) [reply Parsers/ASimAuthentication/ProductParsers/AuthenticationMicrosoftAD4IoT.yaml:8]: Fixing

**Pull request merged**